app/build.gradle

```diff
@@ -130,58 +129,14 @@ tasks.register('codeCoverage', JacocoReport) {
             }
     )
 
-    sourceDirectories.setFrom(fileTree(
-            dir: project.projectDir,
-            includes: [
-                    "src/main/java/**",
-                    "src/main/kotlin/**"
-            ]))
-
-    executionData.setFrom(fileTree(
-            dir: project.buildDir,
-            includes: [
-                    "**/*.exec",
-                    "**/*.ec"
-            ]))
-
-}
-
-tasks.register('instrumentationCodeCoverage', JacocoReport) {
-
-    reports {
-        html.required.set(true)
-        xml.required.set(true)
-    }
-
-    classDirectories.setFrom(
-            fileTree(project.buildDir) {
-                include("**/tmp/kotlin-classes/debug/**")
-                exclude(
-                        '**/R.class',
-                        '**/R$*.class',
-                        '**/BuildConfig.*',
-                        "**/*Application.*",
-                        '**/Manifest*.*',
-                        '**/*Test*.*',
-                        '**/android/**/*.*',
-                        '**/androidx/**/*.*',
-                        '**/airbnb/**/*.*',
-                        "**/di/**/*.*",
-                        '**/*Dagger*.*',
-                        "**/*Screen*"
```

Comment:
**out-of-scope**: We should probably adjust now, those exclude. I.e. 
- `"**/*Application.*",` shuldn't be excluded, because we test application
- `"**/di/**/*.*"` shuldn't be excluded, because we test it
- `"**/*Screen*" shuldn't be excluded, because we test it
- `'**/*Dagger*.*',` - we don't use dagger ;) 
-  `'**/android/**/*.*`, `**/androidx/**/*.*` and `'**/*Test*.*'` are excluded, but doesn't make sense, because those files aren't included in `"**/tmp/kotlin-classes/debug/**"`

But: `ShowkaseRoot`, `ShowkaseRootCodegen`  and `ShowkaseRootShowkaseExcetionsFunctionsCodegenKt` could be excluded from the report,

*out-of-scope*: In the report, files from the `components` module (com.appunite.loudius.components.components) aren't included.

Reply:
Task added!